alphastats/DataSet.py

```diff
@@ -94,6 +94,21 @@ def __init__(
         self.sample: str = sample
         self.preprocessing_info: Dict = preprocessing_info
 
+        self._gene_name_to_protein_id_map = (
+            {
+                k: v
+                for k, v in dict(
+                    zip(
+                        self.rawinput[self._gene_names].tolist(),
+                        self.rawinput[self.index_column].tolist(),
+                    )
+                ).items()
```

Comment:
This is not necessarily unique, and should ideally raise an error in some of our test-data sets that contain isoform ids. E.g. TPM1 occurs 5 times in testfiles/maxquant/proteinGroups.txt with different base Protein IDs.